lib/modules/styleTweaks.js

```diff
@@ -753,5 +772,33 @@ modules['styleTweaks'] = {
 				animation: none !important;	\
 			}	\
 			');
-	}
+	},
+	postTitleCapitalization: function() {
+		switch (this.options.postTitleCapitalization.value) {
+			case 'title':
+				RESUtils.addCSS('.entry a.title { text-transform: capitalize; }');
+				break;
+			case 'sentence':
+				RESUtils.addCSS('\
+					.entry a.title { text-transform: lowercase; display: inline-block; }	\
+					.entry a.title::first-letter { text-transform: uppercase; }	\
+					');
+				break;
+			case 'lowercase':
+				RESUtils.addCSS('.entry a.title { text-transform: lowercase; }');
+				break;
+			case 'none': // fall through
+			default:
+				// do nothing
+		}
+
+		if (this.options.postTitleCapitalization.value === 'sentence') {
+			RESUtils.watchForElement('siteTable', modules['showImages'].findAllImages);
+		}
+
+		function segregateFirstLetter(ele) {
```

Comment:
whoops.  yeah, this was for Sentence Case but then I figured out inline-block worked.